src/pymor/operators/constructions.py

```diff
@@ -82,34 +82,34 @@ def apply(self, U, ind=None, mu=None):
             R.axpy(c, op.apply(U, ind=ind, mu=mu))
         return R
 
-    def apply2(self, V, U, U_ind=None, V_ind=None, mu=None, product=None):
+    def pairwise_apply2(self, V, U, U_ind=None, V_ind=None, mu=None, product=None):
```

Comment:
That seems to be git's fault:
https://github.com/pymor/pymor/blob/b619aac59e56a38fce19a4a82faf45fbed665ada/src/pymor/operators/constructions.py